src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int lotteryCount = 3;
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)
Don’t forget how to name constants according to style guide (the same with naming of enum values)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names